delta/delta_1.1.2_geometry_altitude.post.sql

```diff
@@ -0,0 +1,1401 @@
+     FULL JOIN reach_parts s2 ON s1.gid = (s2.gid - 1) AND s1.fk_wastewater_networkelement::text = s2.fk_wastewater_networkelement::text
+     ORDER BY COALESCE(s1.fk_wastewater_networkelement, s2.fk_wastewater_networkelement), COALESCE(s1.pos, 0::double precision)
+   ) AS rr
+   ON rr.reach_obj_id = re.obj_id
+   LEFT JOIN qgep_od.wastewater_networkelement ne ON ne.obj_id = re.obj_id
+   LEFT JOIN qgep_od.channel ch ON ch.obj_id = ne.fk_wastewater_structure
+   LEFT JOIN qgep_vl.reach_material mat ON re.material = mat.code
+
+   UNION
+
+   SELECT
+     connectors.obj_id AS obj_id,
+     'special_structure' AS type,
+     NULL AS depth,
+     ST_Length( progression_geometry ) AS length_calc,
+     ST_Length( progression_geometry ) AS length_full,
+     from_obj_id,
+     to_obj_id,
+     from_obj_id AS from_obj_id_interpolate,
+     to_obj_id AS to_obj_id_interpolate,
+     0 AS from_pos,
+     1 AS to_pos,
+     bottom_level,
+     NULL AS usage_current,
+     NULL AS material,
+     progression_geometry,
+     progression_geometry AS detail_geometry
+
+   FROM
+   (
+     SELECT
+     wn_from.obj_id AS obj_id,
+     wn_from.obj_id AS from_obj_id,
+     rp_from.obj_id AS to_obj_id,
+     wn_from.bottom_level AS bottom_level,
+     ST_LineFromMultiPoint( ST_Collect(wn_from.situation_geometry, rp_from.situation_geometry ) ) AS progression_geometry
+     FROM qgep_od.reach
+       LEFT JOIN qgep_od.reach_point rp_from ON rp_from.obj_id = reach.fk_reach_point_from
+       LEFT JOIN qgep_od.wastewater_node wn_from ON rp_from.fk_wastewater_networkelement = wn_from.obj_id
+     WHERE
+       reach.fk_reach_point_from IS NOT NULL
+       AND
+       wn_from.obj_id IS NOT NULL
+
+     UNION
+
+     SELECT
+       wn_to.obj_id AS obj_id,
+       rp_to.obj_id AS from_obj_id,
+       wn_to.obj_id AS to_obj_id,
+       wn_to.bottom_level AS bottom_level,
+       ST_LineFromMultiPoint( ST_Collect(rp_to.situation_geometry, wn_to.situation_geometry ) ) AS progression_geometry
+     FROM qgep_od.reach
+       LEFT JOIN qgep_od.reach_point rp_to ON rp_to.obj_id = reach.fk_reach_point_to
+       LEFT JOIN qgep_od.wastewater_node wn_to ON rp_to.fk_wastewater_networkelement = wn_to.obj_id
+     WHERE
+       reach.fk_reach_point_to IS NOT NULL
+     AND
+       wn_to.obj_id IS NOT NULL
+   ) AS connectors
+   LEFT JOIN qgep_od.wastewater_networkelement ne ON ne.obj_id = connectors.obj_id
+ ) AS parts
+WHERE GeometryType(progression_geometry) <> 'GEOMETRYCOLLECTION';
+
+
+
+
+REFRESH MATERIALIZED view qgep_od.vw_network_node;
+REFRESH MATERIALIZED view qgep_od.vw_network_segment;
+
+
+
+
+-----------------------------------------------
+-- View: qgep_od.vw_qgep_reach and rules and triggers
+-----------------------------------------------
+
+CREATE OR REPLACE VIEW qgep_od.vw_qgep_reach AS
+
+/* WITH active_maintenance_event AS (
+  SELECT me.obj_id, me.identifier, me.active_zone, mews.fk_wastewater_structure FROM qgep_od.maintenance_event me
```

Comment:
Was a test? Can you remove it?

Reply:
I can remove it. I took this comment from the original file where this view is created https://github.com/QGEP/datamodel/blob/master/view/vw_qgep_reach.sql#L6 but I agree it does not need to be in the delta as well.